scripts/common.sh

```diff
@@ -1,19 +1,32 @@
 #!/bin/bash
-# Configuration helper
+# Common functions
 
+function generate_key {
+    if [ ! -f "$1" ]; then
+        openssl genrsa -out $1 2048
```

Comment:
Should keysize be a var so it's easier to identify and change in the future?

Reply:
possibly, that and key type could be, but they’re also constrained by the yubikeys

Reply:
You can comment that constrainton the var though?